src/api/transaction/sign.js

```diff
@@ -48,12 +48,14 @@ function computeSignature(txJSON, keypair) {
 }
 
 function sign(txJSON: {Account: string; SigningPubKey: string,
-    TxnSignature: string}, secret: string):
+    TxnSignature: string}, secret: string, isRegular = false):
```

Comment:
Ok, let's do that then to keep things simple. Do you know how to validate a secret by itself?